cmd/gaiad/genaccounts.go

```diff
@@ -119,11 +119,21 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			}
 
 			authGenState := auth.GetGenesisStateFromAppState(cdc, appState)
+			bankGenState := bank.GetGenesisStateFromAppState(depCdc, appState)
```

Comment:
aren't these changes on `master` already?

Reply:
I believe they are, this includes master.